order/src/main/java/nl/softwarestrijders/waiter/order/adapters/http/OrderController.java

```diff
@@ -0,0 +1,68 @@
+package nl.softwarestrijders.waiter.order.adapters.http;
+
+import nl.softwarestrijders.waiter.order.adapters.http.dto.ModifyProductOnOrderDto;
+import nl.softwarestrijders.waiter.order.adapters.http.dto.CreateOrderDto;
+import nl.softwarestrijders.waiter.order.core.application.CommandHandler;
+import nl.softwarestrijders.waiter.order.core.application.QueryHandler;
+import nl.softwarestrijders.waiter.order.core.domain.Order;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/order")
+public class OrderController {
+    private final QueryHandler queryHandler;
+    private final CommandHandler commandHandler;
+
+    public OrderController(QueryHandler queryHandler, CommandHandler commandHandler) {
+        this.queryHandler = queryHandler;
+        this.commandHandler = commandHandler;
+    }
+
+    @PostMapping
+    public Order createOrderById(@RequestBody CreateOrderDto dto) {
+        return this.commandHandler.handleCreateOrder(dto.customerId());
+    }
+
+    @PostMapping("/{id}/product")
+    public Order addProductToOrder(
+            @PathVariable UUID id,
+            @RequestBody ModifyProductOnOrderDto dto) {
+        return this.commandHandler.handleAddProductToOrder(id, dto.productId(), dto.amount());
+    }
+
+    @DeleteMapping("/{id}/product")
```

Comment:
nit (maybe this is better, idk):
```suggestion
    @DeleteMapping("/{id}/product/{productId}")
```